app/assets/stylesheets/hotsheet/application.css

```
@@ -84,3 +91,58 @@ table {
     width: 100%;
   }
 }
+
+#flash-container {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  top: 1rem;
+  gap: 1rem;
+  justify-content: center;
+  width: calc(100% - var(--sidebar-width) - var(--main-padding-x) * 2);
+
+
+  .flash {
+    border-radius: 0.4rem;
+    padding: 0.75rem 1.25rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &.success {
+      background-color: rgb(from var(--success-color) r g b / 80%);
+    }
+
+    &.alert {
+      background-color: rgb(from var(--alert-color) r g b / 80%);
+    }
+
+    &.notice {
+      background-color: rgb(from var(--notice-color) r g b / 80%);
+    }
+
+    &.error {
+      background-color: rgb(from var(--error-color) r g b / 80%);
+    }
+
+    button.close {
```

```suggestion
    .btn-close {
```
I think I'd prefer strict usage of css classes

---

open